src/main/java/pl/stormit/tinyurl/service/IpLocalizationService.java

```diff
@@ -0,0 +1,42 @@
+package pl.stormit.tinyurl.service;
+
+import com.maxmind.geoip2.DatabaseReader;
+import com.maxmind.geoip2.exception.AddressNotFoundException;
+import com.maxmind.geoip2.exception.GeoIp2Exception;
+import com.maxmind.geoip2.model.CityResponse;
+import org.springframework.stereotype.Service;
+import pl.stormit.tinyurl.dto.UrlAnalyticsLocalizationDto;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.InetAddress;
+
+@Service
+public class IpLocalizationService {
+
+    public UrlAnalyticsLocalizationDto getIpLocalization(String addressIp) {
+
+        InetAddress ipAddress;
+        UrlAnalyticsLocalizationDto setLocalization = new UrlAnalyticsLocalizationDto(null, null, null);
+
+        try {
+            ipAddress = InetAddress.getByName(addressIp);
+
+            DatabaseReader database = new DatabaseReader.
+                    Builder(new File("src/main/resources/localizationdb/GeoLite2-City.mmdb")).build();
```

Comment:
Looks like a perfect candidate for a constant.

Reply:
I corrected it.